### DISCUSSION TITLE
Idea, allow this add-on to work with a remote database

### DISCUSSION THREAD

**Discussion by erikpendragon** [hassio-addons/addon-phpmyadmin#31] [Ideas]
i have MariaDB install on a different sever then my Home Assistant and i would like to be able to customize this add-on to use an external database.

**Comment by frenck** [comment]
This has opted before, however, I'm not really planning on adding that.

The main reason is that this add-on provides a simple, configuration free, experience which I value highly.

**Reply by erikpendragon** [reply]
How about having the default be of no configuration is specified it looks internal, if it is defined then use a external db. 
better yet have it use existing the recorder link defined in configuration.yaml, that way this add-on can stay config free.

**Reply by frenck** [reply]
That is not how it works. The UI will still show the options.
